webpack.config.js

```diff
@@ -7,10 +7,10 @@ const NODE_MODULES = process.env.NODE_MODULES || 'node_modules';
 module.exports = {
     entry: {
         newsroom_js: './assets/index.js',
-        settings_js: ['babel-polyfill', './assets/settings/index.js'],
-        user_profile_js: ['babel-polyfill', './assets/user-profile/index.js'],
+        settings_js: ['babel-polyfill', './assets/settings/index.js', './assets/notifications/index.js'],
+        user_profile_js: ['babel-polyfill', './assets/user-profile/index.js', './assets/notifications/index.js'],
         newsroom_css: './assets/style.js',
-        wire_js: ['babel-polyfill', './assets/wire/index.js'],
+        wire_js: ['babel-polyfill', './assets/wire/index.js', './assets/notifications/index.js'],
```

Comment:
would rather create a new entrypoint for notifications and add it to layout, any reason to combine it?

Reply:
No good reason, let me try that.